Automattic/jetpack
# Boost: Update photon compatibility

> **Warning**
> This is a follow-up to #29561. So, it should not be merged before the original PR is merged.


## Proposed changes:
* Do not load photon from jetpack if Boost is serving Image CDN package
* Sync status between Boost's image-cdn and photon

### Other information:


## Jetpack product discussion
None

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
* Activate Boost from this branch alongside Jetpack
* Make sure **Jetpack -> Settings -> Performance -> "Speed up image load times"** syncs with **Boost -> "Image CDN"** when you toggle the status on either place
* Make sure, photon inside Jetpack does not load if Boost has Image CDN enabled. There is no visible way to verify this as the functionality is almost same between Image CDN and Photon, But, you can try hacking by adding `wp_die()` or something in the photon file in Jetpack.
